internal/command/generate.go

```diff
@@ -210,8 +210,21 @@ arguments.
 			}
 		}
 
+		const scoreFilename = "score.yaml"
+		_, issue := os.Stat(scoreFilename)
```

Comment:
```suggestion
		_, err := os.Stat(scoreFilename)
```
Better to follow common golang nomenclature